test/dif_pe_examples/vp/vp_general.json

```diff
@@ -32,7 +32,7 @@
     {
       "comment": "IN REALWORLD VPs, THIS WILL BE A BIG UGLY OBJECT INSTEAD OF THE DECODED JWT PAYLOAD THAT FOLLOWS",
       "vc": {
-        "@context": "https://www.w3.org/2018/credentials/v1",
```

Comment:
Will not disapprove the PR on this change for now, but we really need to add back the generic VC @context together with the specific context for the type everywhere.